server/db/db-connection.js

```diff
@@ -0,0 +1,9 @@
+const { Pool } = require('pg');
+
+console.log(process.env.DB_URI)
+
+const db = new Pool ({
+    connectionString: process.env.DB_URI,
+});
+
+module.exports = db;
```

Comment:
best practice is to leave an empty line as the very last line of the file. Consider adding an empty line after this line

Reply:
Thank you!